xml/System/GCMemoryInfo.xml

```diff
@@ -243,7 +244,7 @@ The memory after `OBJ_D` is not considered part of the `FragmentedBytes` but is
       <Docs>
         <summary>Gets the high memory load threshold when the last garbage collection occurred.</summary>
         <value>The high memory load threshold, in bytes, when the last garbage collection occurred.</value>
-        <remarks>To be added.</remarks>
+        <remarks>Most of the GC performance heuristics are based on per-process measurements. However, to avoid paging, GC is aware of the global physical memory load on the machine. GC recognizes a certain memory load percentage as a "high memory load situation". When the memory load percentage is over that threshold, GC becomes more aggressive and will do more full blocking GCs if necessary to reduce the heap size.</remarks>
```

Comment:
```suggestion
        <remarks>Most of the GC performance heuristics are based on per-process measurements. However, to avoid paging, GC is aware of the global physical memory load on the machine or VM or in the container. GC recognizes a certain memory load percentage as a "high memory load situation". When the memory load percentage is over that threshold, GC becomes more aggressive and will do more full blocking GCs if necessary to reduce the heap size.</remarks>
```